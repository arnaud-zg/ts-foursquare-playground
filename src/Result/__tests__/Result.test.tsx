import React from 'react'
import renderer from 'react-test-renderer'
import { Result } from '../'

describe('Component/Result', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Result />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
