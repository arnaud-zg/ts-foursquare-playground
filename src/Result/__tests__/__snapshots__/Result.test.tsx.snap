// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component/Result renders correctly for action: RESOLVE_GET_LIFE 1`] = `
<article
  className="flex-1 border-b"
>
  <div
    className="h-full border-l-2 bg-grey-lightest border-indigo"
  >
    <header
      className="flex justify-between items-center p-5 pl-8 pr-8 cursor-pointer select-none"
    >
      <span
        className="text-indigo font-thin text-xl"
      >
        Result
      </span>
    </header>
    <div
      className="pl-8 pr-8 pb-5 text-grey-darkest select-none"
    >
      <div>
        <div>
          <h3>
            Life
          </h3>
          <code>
            <pre>
              {"life":false,"credentials":{"clientId":"","clientSecret":""}}
            </pre>
          </code>
        </div>
        <div
          className="mt-4"
        >
          <h3>
            Credentials
          </h3>
          <code>
            <pre>
              {"clientId":"","clientSecret":""}
            </pre>
          </code>
        </div>
      </div>
    </div>
  </div>
</article>
`;

exports[`Component/Result renders correctly for action: RESOLVE_PUT_CREDENTIALS 1`] = `
<article
  className="flex-1 border-b"
>
  <div
    className="h-full border-l-2 bg-grey-lightest border-indigo"
  >
    <header
      className="flex justify-between items-center p-5 pl-8 pr-8 cursor-pointer select-none"
    >
      <span
        className="text-indigo font-thin text-xl"
      >
        Result
      </span>
    </header>
    <div
      className="pl-8 pr-8 pb-5 text-grey-darkest select-none"
    >
      <div>
        <div>
          <h3>
            Life
          </h3>
          <code>
            <pre>
              {"life":false,"credentials":{"clientId":"","clientSecret":""}}
            </pre>
          </code>
        </div>
        <div
          className="mt-4"
        >
          <h3>
            Credentials
          </h3>
          <code>
            <pre>
              {"clientId":"","clientSecret":""}
            </pre>
          </code>
        </div>
      </div>
    </div>
  </div>
</article>
`;
